webrecorder/webrecorder/models/usermanager.py

```diff
@@ -460,7 +460,7 @@ def remove_from_mailing_list(self, email):
                 print('Removing from mailing list failed:', e)
 
     def get_session(self):
-        return request.environ['webrec.session']
+        return request.environ.get('webrec.session')
```

Comment:
Hm, I am curious about this, when would the session be missing?
Is this for the command-line use case?

Reply:
Yes, I had some trouble getting your command-line tool to work properly when trying to delete an user.